### PR TITLE
Integration of RHHC: start using the RHHC components to render the CMS pages and the articles with their modules

### PR DESCRIPTION
**Pull request by Jokinen**
## Description :sparkles:

This PR is meant to be a PoC example of how the RHHC library can be used to display headless-cms pages.

- Implements component that are available from RHHC and deprecates them from this project
- PoC that shows that the RHHC library is Server/Client compatible
- PoC shows that the RHHC library is Next.js compatible

HCRC-6

### CODE REVIEW

**Comment by nikomakela** [comment]
This PR now includes the first implementation of the use of HCRC-library:

1.  The translations are configured.
2. ArchivePage -component is used for subpage search.
3. Collections (with Grid-style) are implemented for Pages and Articles.
4. The CMS Pages are routed via cms-page -path and the CMS Articles are routed via cms-article -path.
5. The Page Sidebar is implemented.
6. Some server side rendering is taken care of with the HCRC.

image

image

**Review by nikomakela — APPROVED** [review]
Not perfect, but it must be integrated at some point. At this point it gives more than takes :)

File: src/headless-cms/components/utils.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        (item) => (
+          <Card
+            key={`card-${item.id}`}
+            {...item}
+            clampText={true}
+            withShadow={true}
+            hasLink={true}
+            url={getCollectionItemUrl(item)}
+            // imageLabel={item.featuredImage?.node?.title ?? ''}
+            // imageUrl={item.featuredImage?.node?.mediaItemUrl ?? ''}
```
Comment (tuomas777) [inline src/headless-cms/components/utils.tsx:37]: Left intentionally?

File: public/locales/sv/common.json [hunk 2]
```diff
@@ -71,5 +72,19 @@
...
+    "loadMore": "Ladda mera",
+    "showAll": "Visa alt",
+    "next": "Nästa",
+    "previous": "Föregående"
+  },
+  "menu": {
+    "menuToggleAriaLabel": "öppna menyn"
+  },
+  "breadcrumb": {
+    "breadcrumbNavigationLabel": "brödsmula navigation",
```
Comment (tuomas777) [inline public/locales/sv/common.json:86]: This is capitalized in the other languages.

**Review by tuomas777 — APPROVED** [review]
Looks good AFAICT 👍

File: src/headless-cms/components/utils.tsx (hunk 1 above)
Comment (nikomakela) [reply src/headless-cms/components/utils.tsx:37]: I thought those might give some hint for next refactoring, so I left them there. Some refactoring and further HCRC-lib integration is definitely needed in future.

**Pull request merged**